lms/djangoapps/certificates/signals.py

```diff
@@ -161,12 +164,43 @@ def _listen_for_enrollment_mode_change(sender, user, course_key, mode, **kwargs)
             return False
 
 
+def _determine_producer_config_for_signal_and_topic(signal, topic):
+    """
+    Utility method to determine the setting for the given signal and topic in EVENT_BUS_PRODUCER_CONFIG
+
+    Records to New Relic for later analysis.
+
+    Parameters
+        signal (OpenEdxPublicSignal): The signal being sent to the event bus
+        topic (string): The topic to which the signal is being sent (without environment prefix)
+
+    Returns
+        True if the signal is enabled for that topic in EVENT_BUS_PRODUCER_CONFIG
+        False if the signal is explicitly disabled for that topic in EVENT_BUS_PRODUCER_CONFIG
+        None if the signal/topic pair is not present in EVENT_BUS_PRODUCER_CONFIG
+    """
+    event_type_producer_configs = getattr(settings, "EVENT_BUS_PRODUCER_CONFIG",
+                                          {}).get(signal.event_type, {})
+    topic_config = event_type_producer_configs.get(topic, {})
+    topic_setting = topic_config.get('enabled', None)
+    set_custom_attribute(f'producer_config_setting_{topic}_{signal.event_type}',
+                         topic_setting if topic_setting is not None else 'Unset')
+    return topic_setting
+
+
 @receiver(CERTIFICATE_CREATED)
 def listen_for_certificate_created_event(sender, signal, **kwargs):  # pylint: disable=unused-argument
     """
     Publish `CERTIFICATE_CREATED` events to the event bus.
     """
+    # temporary: defer to EVENT_BUS_PRODUCER_CONFIG if present
+    producer_config_setting = determine_producer_config_for_signal_and_topic(CERTIFICATE_CREATED,
+                                                                             'learning-certificate-lifecycle')
+    if producer_config_setting is True:
```

Comment:
Suggestion: We could tighten this up with a shorter name and inlining. I know it's meant to be temporary code, so this isn't blocking feedback, but it would be easier to review this way.

```suggestion
    if producer_configured_for(CERTIFICATE_CREATED, 'learning-certificate-lifecycle') is True:
```

I'm guessing the use of `is True` is a safeguard against forgetting to make a call to `derived_collection_entry`?

Reply:
(But I've finished reviewing all these blocks, so it's probably not worth changing.)